packages/xo-web/src/xo-app/backup/index.js

```diff
@@ -16,6 +16,10 @@ const DeprecatedMsg = () => (
   <div>
     <em>{_('backupDeprecatedMessage')}</em>
     <br />
+    <a href='https://xen-orchestra.com/blog/migrate-backup-to-backup-ng/'>
+      {_('backupMigrationLink')}
+    </a>
+    <br />
```

Comment:
IMHO, it would be more obvious by using an [warning alert](https://v4-alpha.getbootstrap.com/components/alerts/).